### ISSUE TITLE
Home page

### ISSUE THREAD

**Issue by PlayingKarrde** [PlayingKarrde/gameOS#42]
Add a starting home page that contains the recently played items along with most played and favourites.

Blocked by Pegasus collections (update needed)

**Comment by ecleese** [comment]
I'd love to see a home page added. Is this still on hold due to a Pegasus limitation? Was an issue ever created asking for collections support?

**Comment by PlayingKarrde** [comment]
This is the next big ticket item on my list actually. I don't believe I'm blocked by Pegasus anymore.
